prowler/lib/outputs/html.py

```diff
@@ -222,13 +224,14 @@ def fill_html(file_descriptor, finding):
                 <td>{finding.region}</td>
                 <td>{finding.check_metadata.CheckTitle}</td>
                 <td>{finding.resource_id.replace("<", "&lt;").replace(">", "&gt;").replace("_", "<wbr>_")}</td>
-                <td>{str(finding.resource_tags)}</td>
+                <td>{unroll_list(finding.resource_tags)}</td>
```

Comment:
Can we separate each element using a comma?

Reply:
We separate it with pipes (|)

Reply:
Done